modules/custom/inope_policy_review/inope_policy_review_handler_note_status_boolean.inc

```diff
@@ -10,18 +10,21 @@
  * Allows for display of approved / changes requested.
  *
  * Definition terms:
- *   - output formats: An array where the first entry is displayed on boolean true
- *      and the second is displayed on boolean false. An example for sticky is:
- *      @code
- *      'output formats' => array(
- *        'sticky' => array(t('Sticky'), ''),
- *      ),
- *      @endcode
+ *   - output formats: An array where the first entry is displayed on boolean
+ *      true and the second is displayed on boolean false.
  *
  * @ingroup inope_policy_review
  */
 class inope_policy_review_handler_note_status_boolean extends views_handler_field_boolean {
-  function option_definition() {
+  /**
+   * Override the default boolen field handlers default options.
+   *
+   * We are changing the default option to approved-requested created in init.
+   *
+   * @return array $options
+   *   Sets the default options to be used in the init method.
+   */
+  protected function option_definition() {
```

Comment:
So this is the line that is causing the issue -- why did you choose protected here?

It's important to understand how implementing classes work with this PHP OOP stuff. If you are overriding existing classes - changing the scope can cause problems.